src/board.js

```diff
@@ -191,6 +191,9 @@ class BoardUI {
       new ButtonUI(this.svg.querySelector("#ButtonA"), "A"),
       new ButtonUI(this.svg.querySelector("#ButtonB"), "B"),
     ];
+    this.pins = Array(33)
+    this.pins[MICROBIT_HAL_PIN_FACE] = 
+      new PinUI(this.svg.querySelector("#Logo"), "pin_logo"),
```

Comment:
I suspect this needs improving. Looks inside an array by index. `MICROBIT_HAL_PIN_FACE = 19`.

Reply:
I think this is fine for now